build.gradle.kts

```diff
@@ -5,11 +5,12 @@ plugins {
     buildsrc.conventions.lang.`kotlin-multiplatform-jvm`
     buildsrc.conventions.lang.`kotlin-multiplatform-js`
     buildsrc.conventions.lang.`kotlin-multiplatform-native`
+    buildsrc.conventions.`git-branch-publish`
 }
 
 group = "org.snakeyaml"
 version = "2.7-SNAPSHOT"
```

Comment:
Let's discuss separately how we should version the library. There's also another option, to start from 0.x.

Reply:
yes, that crossed my mind too 👍. Also, potentially an entirely new group & name?